frontend/taipy-gui/base/src/app.ts

```diff
@@ -101,6 +103,16 @@ export class TaipyApp {
         this._onReload = handler;
     }
 
+    get onWsMessage() {
+        return this._onWsMessage;
+    }
+    set onWsMessage(handler: OnWsMessage | undefined) {
+        if (handler !== undefined && handler?.length !== 3) {
+            throw new Error("onWsMessage() requires three parameters");
+        }
+        this._onWsMessage = handler;
```

Comment:
I wonder if you should `bind` the function so that this will be defined and you don't have to pass taipyApp as first parameter ?

Reply:
You could also have a default implementation that does nothing and would allow you to not check the existence before each call...

Reply:
i think I can replace the get method with a default function that can also check for its existence. the method will also bypass the passing of the app